charts/backstage/values.yaml

```diff
@@ -47,6 +47,9 @@ upstream:
     # create multiple DBs. Since Backstage requires by default 5 different DBs, we
     # can't accommodate that properly.
     appConfig:
+      auditLog:
+        rotateFile:
+          enabled: true
```

Comment:
I guess docs also need to be updated accordingly.. It currently reads: `Audit logging to a rotating file is disabled by default.`
https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/getting_started_with_red_hat_developer_hub/assembly-audit-log#ref-audit-log-file-rotation-overview_assembly-audit-log

Reply:
But we (RHDH Product) only support Helm and the Operator as installation methods, no? And that's what is covered in the current docs.

Reply:
> Well, technically within the RHDH container image, audit logging is disabled by default, we are just enabling it via the helm and operator installation methods. 

Or we can explicitly state in the docs that the Helm Chart and the Operator enable it by default. Otherwise, it can be confusing, IMO. For example, I found another occurrence where it reads: `Set enabled to true to use audit log file rotation. By default, it is set to false.`
But this can be addressed in a specific docs issue/PR.

/lgtm

Reply:
I will mark the doc update in the JIRA